systemTests/test-scenarios/conftest.py

```diff
@@ -0,0 +1,22 @@
+import os
+import shutil
+import pytest
+
+@pytest.fixture(scope="session", autouse=True)
+def set_up_system_test_environment(request):
+    print("setting up a system test environment")
+
+    # clear out the old programs
+    execution_space = '../execution_space'
+    if (os.path.isdir(execution_space)):
+        shutil.rmtree(execution_space)
+
+    # copy over the src
+    os.makedirs(execution_space)
+    shutil.copytree('../../src', execution_space + '/src')
```

Comment:
why do you copy over the source folder? I'm not sure anything is running out of here so I don't think it's needed.

We need to work out how to deploy the system so that the system tests can be run, I reckon this will mean building it and then updating the library path to include our build folder.

Reply:
I was mirroring the layout from the daaas-dms version but you are probably right! I'll remove the `shutil.copytree('../../src', execution_space + '/src')` line